internal/cmd/logout.go

```diff
@@ -2,31 +2,76 @@ package cmd
 
 import (
 	"errors"
+	"fmt"
+	"os"
 
 	"github.com/spf13/cobra"
 
 	"github.com/infrahq/infra/internal/logging"
 )
 
 func newLogoutCmd() *cobra.Command {
-	var purge bool
+	var (
```

Comment:
Might want to consider using a struct to hold the options since you're passing them all to `logout`

Reply:
agreed, although this should be standardized for all cmds in a single pr (some uses parseOptions, some don't, some use vars). I believe Daniel was discussing this somewhere too

Reply:
Ya, I think these can be an options struct, `logoutOptions`